<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1012</width>
    <height>763</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QCalendarWidget" name="calendarWidget"/>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>订单</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_23">
          <item>
           <widget class="QLabel" name="label_22">
            <property name="text">
             <string>下单时间</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="send_date"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_18">
          <item>
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>订单类型</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="order_type">
            <item>
             <property name="text">
              <string>期权合约</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>期权标的</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_19">
          <item>
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>标的ID</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="underlying_id"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_20">
          <item>
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string>合约ID</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="contract_id"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_21">
          <item>
           <widget class="QLabel" name="label_20">
            <property name="text">
             <string>开平标志</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="position_effect">
            <item>
             <property name="text">
              <string>开仓</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>平仓</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_22">
          <item>
           <widget class="QLabel" name="label_21">
            <property name="text">
             <string>买卖方向</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="side">
            <item>
             <property name="text">
              <string>买入</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>卖出</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_25">
        <item>
         <widget class="QLabel" name="label_24">
          <property name="text">
           <string>委托量</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="volume">
          <property name="maximum">
           <number>100000</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="add_order">
          <property name="text">
           <string>添加</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>828</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="treeWidget">
     <column>
      <property name="text">
       <string>委托下单时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">证券ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>合约ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>开平标志</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>买卖方向</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>订单类型</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>委托量</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
  <action name="delete_order">
   <property name="text">
    <string>删除订单</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
